dev/com.ibm.ws.microprofile.config.1.2_fat_tck/build.gradle

```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+	
+dependencies {
+  requiredLibs 'org.hamcrest:hamcrest-all:1.3'
```

Comment:
Maven co-ordinates of libraries to add to /lib/ dir from where the Arquillian extension can shrinkwrap them in test wars.